src/routes/Wallets/routes/Addresses/sagas/walletsAddresses.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import { delay } from 'redux-saga'
+import { all, call, put, select, takeEvery } from 'redux-saga/effects'
+
+import config from 'config'
+import web3 from 'services/web3'
+import walletsWorker from 'workers/wallets'
+import { selectWallets, selectWalletsAddresses } from 'store/stateSelectors'
+import * as wallets from 'routes/Wallets/modules/wallets'
+
+import * as walletsAddresses from '../modules/walletsAddresses'
+
+function* openView(): Saga<void> {
+  const { items, activeWalletId } = yield select(selectWallets)
+  const { iteration } = yield select(selectWalletsAddresses)
+
+  const startIndex: Index = config.mnemonicAddressesCount * iteration
+  const endIndex: Index = (startIndex + config.mnemonicAddressesCount) - 1
+
+  if (!activeWalletId) {
+    return
+  }
+
+  yield put(walletsAddresses.getMoreRequest(items, activeWalletId, startIndex, endIndex))
```

Comment:
Ok, I understood the problem with this, should be like this

```suggestion
  if (activeWalletId) {
    return  yield put(walletsAddresses.getMoreRequest(items, activeWalletId, startIndex, endIndex))
    }
```